src/schemas/secrets.schema.json

```diff
@@ -1,6 +1,6 @@
 {
   "meta:license": [
-    "Copyright 2018 Adobe. All rights reserved.",
+    "Copyright 2019 Adobe. All rights reserved.",
```

Comment:
I don't think you have to update the year here. The initial claim has been made in 2018, what has been added in 2019 is only incremental.

Reply:
OK